src/common/utils/messages.ts

```diff
@@ -236,3 +252,156 @@ export async function waitOperationIds(
       return exhaustiveCheck(chainType);
   }
 }
+
+export function isReversibleAction(action: Action, messageDirection: MessageDirection) {
+  // @ts-expect-error: ts(2345)
+  if (messageDirection === MessageDirection.HubToSpoke) return REVERSIBLE_HUB_ACTIONS.includes(action);
+  return false;
```

Comment:
Can you simplify to `return messageDirection === MessageDirection.HubToSpoke && REVERSIBLE_HUB_ACTIONS.includes(action)`?

Reply:
Resolved with the merge 343cc32aedefbb51689d5302c9cff3cf97cc1b49